docs/get_started/assembly_line_steps.md

```diff
@@ -1,43 +0,0 @@
----
-id: assembly_line_steps
-title: The Assembly Line process
-sidebar_label: The Assembly Line process
-slug: /get_started/assembly_line_steps
----
-
-:::warning
-We are actively updating the Document Assembly Line workflow and much of the information in this section is outdated.
-:::
-
-The guidance below is for organizations that want to set up their **own** copy of the 
-Assembly Line, not for individual automators.
-
-We think that the most important work we did came not from computer code, but a
-process: the process of automating a legal form, broken down into a series of
-small steps.
-
-## Our Trello boards {#trello}
-We turned that process into a series of
-[Kanban](https://blog.trello.com/kanban-data-nave) boards, and if you want you
-can jump straight to
-[copying](https://help.trello.com/article/802-copying-cards-lists-or-boards) and
-customizing them for your needs:
-
-1. [AssemblyLine - Pre-processing Template Board](https://trello.com/b/Z2Svx3oh/1-assemblyline-pre-processing-template-board#)
-1. [AssemblyLine - Development Template Board](https://trello.com/b/ArfGFbz4/2-assemblyline-development-template-board)
-1. [AssemblyLine - Testing Template Board](https://trello.com/b/nT7yy2Wl/3-assemblyline-testing-template-board)
-
-## Overview
-
-Automating a form involves:
-
-* Research and vetting the substantive content
-* Coding
-* Adding labeled fields to a template document
-* Generating and refining a prototype
-* Testing and getting feedback
-
-## More reading
-
-* [Building your first expert system](https://suffolklitlab.org/legal-tech-class/docs/interview-structure/building-an-app-outline)
-* Lauritsen and Soudakoff, [Keys to a Successful Document Assembly Project](https://static1.squarespace.com/static/571acb59e707ebff3074f461/t/5946f745725e25bf7ad93c9b/1497823045990/keys.pdf)
```

Comment:
The first one is linked prominently on the [Roadmap page](https://assemblyline.suffolklitlab.org/docs/get_started/roadmap). The second is super old. It might inform our documentation, but it's probably not worth including otherwise.